package.json

```diff
@@ -43,9 +43,9 @@
     "@formatjs/intl-numberformat": "^6.2.5",
     "@formatjs/intl-pluralrules": "^4.0.13",
     "@oguzhnatly/react-native-image-manipulator": "^1.0.5",
-    "@onfido/react-native-sdk": "^5.4.0",
+    "@onfido/react-native-sdk": "git+https://github.com/AndrewGable/react-native-sdk.git#0987a560b4f5f076ea595a3fa692c53ed383753f",
```

Comment:
Hack: We should fork this under Expensify org.

Reply:
Done here: https://github.com/Expensify/react-native-sdk/pull/1 - Just waiting for it to be merged.

Reply:
Fixed this!